common/src/main/java/apoc/convert/ConvertUtils.java

```diff
@@ -1,7 +1,6 @@
 package apoc.convert;
 
-import org.neo4j.internal.helpers.collection.Iterators;
-
+import apoc.util.collection.Iterators;
```

Comment:
Not directly related, but we could change rows 20 and 21 with these?
```
        else if (list instanceof Iterable) return Iterables.asList((Iterable)list);
        else if (list instanceof Iterator) return Iterators.asList((Iterator)list);
```

Reply:
Nice catch!